src/graphql/client/graphql-client.ts

```diff
@@ -471,14 +471,16 @@ export class GraphQLClient {
     ids: string[],
     session: string
   ): Promise<void> {
+    const minRefreshedAt = new Date(this.resetLimitMillis).toISOString();
```

Comment:
Hmm do we actually want to make a separate function that we call to add the refreshedAt limit? I realized this is public so changing its behavior like this might be a bad idea

Reply:
it's ok. this applies in a general case as well.

Reply:
basically we should all the same delete conditions we have in the query:
```
const deleteConditions: any = {
        refreshedAt: {_lt: minRefreshedAt},
        ...originFilter,
};
```
Lines 424-426